app/services/symphony_client.rb

```diff
@@ -41,6 +41,17 @@ def session_token
     end
   end
 
+  def checkouts(patron_key)
+    circ_rule_query = 'circulationRule{loanPeriod{periodType{key}}}'
+    item_query = 'item{*,bib{title,author},call{sortCallNumber,dispCallNumber}}'
+
+    response = authenticated_request("/user/patron/key/#{patron_key}", params: {
+      includeFields: "*,circRecordList{*,#{[circ_rule_query, item_query].join(',')}}"
```

Comment:
Can you add these additional fields to the `patron_info` method below (and then remove this method?)